src/ripple/proto/org/xrpl/rpc/v1/xrp_ledger.proto

```diff
@@ -51,4 +52,7 @@ service XRPLedgerAPIService {
   // ledgers. Note, this method has no JSON equivalent.
   rpc GetLedgerDiff(GetLedgerDiffRequest) returns (GetLedgerDiffResponse);
 
+  // Get account info for an account on the XRP Ledger.
+  rpc GetAmmInfo (GetAmmInfoRequest) returns (GetAmmInfoResponse);
+
```

Comment:
Let's not add this. These gRPC methods were for the xpring sdk, which has been abandoned

Reply:
@cjcobb23 I'm working on another AMM PR. I'll incorporate your feedback in this new PR. To sum it up - AMM transactions and GetAmmInfo() should be removed from gRPC.